app/assets/stylesheets/petitions/views/_home.scss

```diff
@@ -87,7 +91,15 @@
   }
 }
 
-.local-to-you h2,
-.start-petition h2 {
-  @include bold-24;
+#local-to-you-heading,
```

Comment:
We should not use the #anchors for headings. Better to use the classes as was previously done.